service/azureconfig/v1/arm_templates/main.json

```diff
@@ -8,12 +8,21 @@
         "description": "Unique ID of this guest cluster."
       }
     },
+    "hostClusterResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Host cluster resource group."
+      }
+    },
     "GiantSwarmTags": {
       "type": "object",
       "defaultValue": {
         "provider": "F80D01C0-7AAC-4440-98F6-5061511962AD"
       }
     },
+    "virtualNetworkName": {
+      "type": "string"
+    },
```

Comment:
Why is there a description on `hostClusterResourceGroup` but not on `virtualNetworkName`?

Reply:
virtualNetworkName is for guest cluster.